sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/CbsTokenProvider.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Authorization;
+using Azure.Messaging.EventHubs.Core;
+using Microsoft.Azure.Amqp;
+
+namespace Azure.Messaging.EventHubs.Amqp
+{
+    /// <summary>
+    ///   Performs the actions needed to generate <see cref="CbsToken" /> instances for
+    ///   authorization within an AMQP scope.
+    /// </summary>
+    ///
+    /// <seealso cref="Microsoft.Azure.Amqp.ICbsTokenProvider" />
+    ///
+    internal sealed class CbsTokenProvider : ICbsTokenProvider
+    {
+        /// <summary>The type to consider a token if it is based on an Event Hubs shared access signature.</summary>
+        private const string SharedAccessSignatureTokenType = "servicebus.windows.net:sastoken";
+
+        /// <summary>The type to consider a token if not based on a shared access signature.</summary>
+        private const string JsonWebTokenType = "jwt";
+
+        /// <summary>;The type to consider a token generated from the associated <see cref="Credential" />.</summary>
```

Comment:
typo: semicolon before text. `;The type...`.

Reply:
Thanks.   I just did the fix on the wrong local branch, since its just a comment I'm going to defer until that PR.